app/src/pages/explorer/Home.vue

```diff
@@ -67,6 +70,11 @@ export default {
       ]
     }
   },
+  computed: {
+    dialogBox () {
+      return this.$store.getters.dialogBox
+    }
+  },
```

Comment:
nit: why not use the `mapGetters` helper?

Reply:
That's accurate. I narrowed the dev to focus on the core dialog box itself. I created ticket #80 for that fix. To avoid conflict it wasn't implemented on this ticket